c/ntdrivers/kbfiltr_false-unreach-call_true-valid-memsafety.i.cil.c

```diff
@@ -2362,6 +2365,7 @@ void stub_driver_init(void)
   return;
 }
 }
+extern unsigned char __VERIFIER_nondet_uchar(void);
```

Comment:
It would be nicer if that were placed with the other declarations, at the top of the file.